src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/DefaultHtmlGeneratorExtensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.Mvc.Rendering;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Mvc.ViewFeatures
+{
+    public static class DefaultHtmlGeneratorExtensions
```

Comment:
Why extend `IHtmlGenerator` in a `public` namespace for this purpose? Suggest we choose one of

1. We don't need methods for these purposes because they are incredibly short helpers than work fine with the code inlined within `FormTagHelper`.
2. We need separate helper methods but don't need to split them from `FormTagHelper` because we aren't extending the HTML helper today or tomorrow. (Yeah, I know @NTaylorMullen said the opposite.)
3. We need separate helpers but not for public consumption. Instead these methods should be regular `static` methods somewhere in `Microsoft.AspNetCore.Mvc.ViewFeatures.Internal`. (We only rarely define or use extension methods. Not justified here.)